docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -0,0 +1,424 @@
+      requests:
+        memory: "64Mi"
+        cpu: "250m"
+      limits:
+        memory: "128Mi"
+        cpu: "500m"
+```
+
+## How Pods with resource requests are scheduled
+
+When you create a Pod, the Kubernetes scheduler selects a node for the Pod to
+run on. Each node has a maximum capacity for each of the resource types: the
+amount of CPU and memory it can provide for Pods. The scheduler ensures that,
+for each resource type, the sum of the resource requests of the scheduled
+Containers is less than the capacity of the node. Note that although actual memory
+or CPU resource usage on nodes is very low, the scheduler still refuses to place
+a Pod on a node if the capacity check fails. This protects against a resource
+shortage on a node when resource usage later increases, for example, during a
+daily peak in request rate.
+
+## How Pods with resource limits are run
+
+When the kubelet starts a Container of a Pod, it passes the CPU and memory limits
+to the container runner: Docker or rkt.
+
+When using Docker:
+
+- The `spec.containers[].resources.requests.cpu` is converted to its core value,
+  which is potentially fractional, and multiplied by 1024. This number is used
+  as the value of the
+  [`--cpu-shares`](https://docs.docker.com/engine/reference/run/#/cpu-share-constraint)
+  flag in the `docker run` command.
+
+- The `spec.containers[].resources.limits.cpu` is converted to its millicore value,
+  multiplied by 100000, and then divided by 1000. This number is used as the value
+  of the [`--cpu-quota`](https://docs.docker.com/engine/reference/run/#/cpu-quota-constraint)
+  flag in the `docker run` command.  he [`--cpu-period`] flag is set to 100000,
+   which represents the default 100ms period for measuring quota usage. The
+   kubelet enforces cpu limits if it is started with the
+  [`--cpu-cfs-quota`] flag set to true. As of Kubernetes version 1.2, this flag
+  defaults to true.
+
+- The `spec.containers[].resources.limits.memory` is converted to an integer, and
+  used as the value of the
+  [`--memory`](https://docs.docker.com/engine/reference/run/#/user-memory-constraints)
+  flag in the `docker run` command.
+
+If a Container exceeds its memory limit, it might be terminated. If it is
+restartable, the kubelet will restart it, as with any other type of runtime
+failure.
+
+A Container might or might not be allowed to exceed its CPU limit for extended
+periods of time. However, it will not be killed for excessive CPU usage.
+
+To determine whether a Container cannot be scheduled or is being killed due to
+resource limits, see the
+[Troubleshooting](#troubleshooting) section.
+
+## Monitoring compute resource usage
+
+The resource usage of a Pod is reported as part of the Pod status.
+
+If [optional monitoring](http://releases.k8s.io/{{page.githubbranch}}/cluster/addons/cluster-monitoring/README.md)
+is configured for your cluster, then Pod resource usage can be retrieved from
+the monitoring system.
+
+## Troubleshooting
```

Comment:
Just wondering in general, is a Concept page the proper place where we should have the associated Troubleshooting section?